doc/dev_manual/image_viewer.rst

```diff
@@ -0,0 +1,501 @@
+.. _ch-image-viewer-operations:
+
++++++++++++++++++++++++++++++
+Ginga Image Viewer Operations
++++++++++++++++++++++++++++++
+
+This chapter describes the operations supported by the basic ginga image
+viewer and how to access them programatically.  For all of these
+API examples we assume the viewer object is contained in variable `v`.
+
+Loading the Viewer
+==================
+
+The image viewer can load data in a number of formats, but all formats
+are wrapped with a class that corresponds to the *model* part of the
+model-view-controller design used by Ginga.
+For more information on creating the wrapper object,
+see :ref:`ch-image-data-wrappers`.
+
+Once you have successfully loaded an image wrapper, you can set it into
+the image viewer::
+
+  >>> v.set_image(img)
+
+
+======================================
+Manipulating the view, programatically
+======================================
+
+The sections below describe how to manipulate the view programatically.
+Many of the API calls simply set a value in the viewer's settings
+object; if so, we note the associated keyword for that setting.
+
+
+Scaling
+=======
+
+In Ginge lexicon, "scaling" is setting a precise scale factor for pixels.
+For example a scale factor of 2.0 means that two pixels are shown in the
+viewer for every one pixel of data.
+
+Ginga allows different scale factors for the X and Y axes.
+
+Set a precise scale for X and Y::
+
+  >>> v.set_scale((0.5, 0.5))
+
+.. note: Corresponds to the viewer setting `scale`.
```

Comment:
`.. note::` (double colon) -- unless you really meant to comment out this line (will not be rendered when commented)

Also, use double-backtick for "scale".